inc/dashboard/namespace.php

```diff
@@ -0,0 +1,436 @@
+<?php
+/**
+ * Dashboard Analytics
+ *
+ * @package aws-analytics
+ */
+
+namespace Altis\Analytics\Dashboard;
+
+use function Altis\Analytics\Blocks\get_views as get_block_views;
+use function Altis\Analytics\Utils\query;
+
+/**
+ * Set up the Dashboard Analytics page.
+ */
+function setup() {
+	add_filter( 'manage_posts_columns', __NAMESPACE__ . '\\remove_default_columns', 10, 2 );
+	add_filter( 'post_row_actions', __NAMESPACE__ . '\\remove_post_row_actions', 10, 2 );
+	add_filter( 'manage_edit-xb_sortable_columns', __NAMESPACE__ . '\\xb_table_sorting' );
+	add_filter( 'request', __NAMESPACE__ . '\\xb_block_column_orderby' );
+	add_filter( 'bulk_actions-edit-xb', '__return_empty_array' );
+	add_filter( 'views_edit-xb', __NAMESPACE__ . '\\render_date_range_links' );
+	add_filter( 'months_dropdown_results', '__return_empty_array' );
+	add_filter( 'manage_xb_posts_columns', __NAMESPACE__ . '\\add_microcopy_to_column_titles', 99 );
+
+	add_action( 'pre_get_posts', __NAMESPACE__ . '\\modify_views_list_query' );
+	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\enqueue_styles' );
+}
+
+/**
+ * Enqueue styles.
+ */
+function enqueue_styles() {
+	wp_enqueue_style( 'xb-insights', plugin_dir_url( __FILE__ ) . '/assets/style.css', [], '18-03-2021', 'screen' );
+}
+
+/**
+ * Remove the default post list columns.
+ *
+ * @param array $columns An array of columns to filter.
+ * @param string $post_type The current post type.
+ *
+ * @return array The filtered columns array.
+ */
+function remove_default_columns( $columns, $post_type ) : array {
+	if ( 'xb' === $post_type ) {
+		unset( $columns['cb'] );
+		unset( $columns['title'] );
+		unset( $columns['altis_publication_checklist_status'] );
```

Comment:
Since there are no (and will be no) bulk actions for items in this list, there's no reason for them to have checkboxes. But I can see if I can figure out how to remove post type support for this.

Reply:
Sorry - should have been more specific, I was referring to the publication checklist. Review comments are generally associated with the last line in what’s shown, bit confusing as we can do multiline comments now.

Reply:
That makes more sense now. 😅